addons/UtilityClasses/RangeSlider.cpp

```diff
@@ -6,13 +6,16 @@
  * 		Heikki Suhonen <heikki.suhonen@gmail.com>
  *
  */
-#include <stdlib.h>
 #include <Window.h>
+#include <stdlib.h>
 
 #include "RangeSlider.h"
```

Comment:
I would include RangeSlider.h first

Reply:
In general, the way I try to do includes is: 

1. corresponding header (like in this case, RangeSlider.h in RangeSlider.cpp
2. ArtPaint includes
3. Haiku includes
4. standard includes

e.g.:
```
#include "BitmapUtilities.h"

// ArtPaint includes
#include "HSPolygon.h"
#include "PixelOperations.h"
#include "UtilityClasses.h"

// Haiku includes
#include <Screen.h>

// standard C/C++ includes
#include <stdlib.h>
```

I am 100% positive they don't all follow this at the moment but this should be the goal imo.

Reply:
As mentioned further down, to appease the dancing compiler fairies, I didn't touch the include-order.